libraries/AP_NavEKF2/AP_NavEKF2.h

```diff
@@ -72,7 +72,11 @@ class NavEKF2 {
     // returns the index of the IMU of the primary core
     // return -1 if no primary core selected
     int8_t getPrimaryCoreIMUIndex(void) const;
-    
+
+    // returns the index of the IMU of the core instance
+    // return -1 if invalid core selected
```

Comment:
I'd rather see us add a getGyroBiasByIMU() function, which returns bool, and a Vector3f reference. It would try to find an EKF core using the given IMU and return its bias